pkg/ccr/base/spec.go

```diff
@@ -857,6 +857,36 @@ func (s *Spec) TruncateTable(destTableName string, truncateTable *record.Truncat
 	return s.DbExec(sql)
 }
 
+func (s *Spec) RenameTable(destTableName string, renameTable *record.RenameTable) error {
+	// rename table may be 'rename table', 'rename rollup', 'rename partition' and 'rename column'
```

Comment:
remove `and 'rename column'` due to https://github.com/selectdb/ccr-syncer/pull/139